Hunting Queries/LAQueryLogs/NewUserQueries.yaml

```diff
@@ -0,0 +1,26 @@
+id: 8c4fb385-98b0-4ef5-b3da-65db0fb22d89
+name: New users running queries
+description: |
+  'This hunting query looks for users who have run queries that have not previously been seen running queries.'
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
Lets use connectorId: "AzureMonitor(Query Audit)" for now, it may change slightly but should be under Azure Monitor.